### ISSUE TITLE
Wifi not working on Mac mini (Late 2012)

### ISSUE THREAD

**Issue by sedir** [BenSova/Patched-Sur#722]
**Describe the bug**
Wifi stopped working.

**To Reproduce**
Installed last update using Patched Sur and the Wifi icon was animating like it would connect to some network, but it keeps like this for a while and then turns gray.

**Expected behavior**
I expected the wifi to work.

**Screenshots**
![image](https://user-images.githubusercontent.com/523134/130070359-a3aa8399-d448-4b33-bc8b-7b16934ee2a7.png)

**Desktop (please complete the following information):**
 - Upgrading from: Big Sur v11.5.0
 - Upgrading to: Big Sur v11.5.1 and then v11.5.2
 - Mac Model: Mac mini (Late 2012)
 - Version v0.2.1 (96)

**Comment by BenSova** [comment]
Did you patch the kexts?

**Comment by sedir** [comment]
Yes, I did it after both updates and even repeated the proccess.

**Comment by rjs360** [comment]
I have the same issue on a Macbook Pro 9,1 (mid-2012).  If this is useful, my upgrade path was as follows: 1. reinstall of original OS (Mountain Lion), 2. El Capitan upgrade, 3. Catalina upgrade, 4. Patched Sur upgrade.  I patched the kexts via the app. Everything is working well except wifi, which can be turned on/off, but does not recognize any networks. Same initial animation upon reboots, then the wifi icon turns grey.

**Comment by develVannessa** [comment]
I always patch the kext by restarting with the USB installer and Patching from the menu. works like a "charm"
